Bottom: 5f255a4f66f9624dfa27e0981987b111fc46d33c
Top:    f1f434afc6a3554ccd496936ca7f79887e95fbe2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c365f9c..ef9d57f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1297,6 +1297,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
